docs/index.md

```diff
@@ -1,3 +1,45 @@
 ## The UW MRO automated focusing system
 
 These pages will describe the automated focusing system being developed by students in UW Astronomy Undergrad Engineering Group (AUEG).
+
+
+
+
+Camera/telescope communication:
+* Determine how to command the focusing mechanism via python.
+  * Write python code to move the focuser.
+* Determine how to command exposures from the camera via python.
+  * Write python code to command an exposure.
+* Determine how much of a focus move is necessary to significantly change the FWHM of an image.
+
+Focus math:
+* Collect test data of previously-acquired focus images.
+* Write python class to determine the average FWHM of an image (photutils?).
+* Write python class to fit a parabola to a set of points with uncertainties (scipy.optimize?).
+
+Putting it all together:
+* Write a script to take exposures while moving the focuser, fit PSFs to the images, and compute the optimum focus.
+* Generate plots to monitor the output of the above.
+
+
+My suggestion of a place to start would be to find a dozen or so images from a focus sequence at MRO, some above, some near, and some below focus. This will give us some test data to play with. We would also want the logbooks that go with those images, so that we can compare with the focus values measured by the observers.
```

Comment:
Maybe reword this line:

"Starting with 10 images at, above, and below focus...."